### DISCUSSION TITLE
Sound issues with Sega ages games.

### DISCUSSION THREAD

**Discussion by MrAnty** [skyline-emu/skyline#1591] [General]
I have tried both Outrun and Thunder force, and they both have the same problem. When booting, the FPS fluctuates, and the sound is slow and stuttery. In game the FPS is a solid 60, and the game plays fine, but the music is slow and stuttery. Has anyone else had this problem? I have a Pixel 4a, which has a Snapdragon 730G. I know it's not top spec, but I can't see that being the issue with these games.

Thanks.

**Comment by Crytonics** [comment]
Some games audio dosen't work or have audio issues. It will be fixed in the future.
